github/Branch.py

```diff
@@ -196,6 +206,10 @@ def edit_protection(
             or required_approving_review_count is not github.GithubObject.NotSet
         ):
             post_parameters["required_pull_request_reviews"] = {}
+            if self._rawData.get("is_organization", False):
```

Comment:
Please add a property to Branch that exposes it and check it here.

Reply:
I wasnt sure if it ought to be exposed as it isn't a native GitHub attribute, but I can certainly do that.